account-kit/react/src/context.tsx

```diff
@@ -38,7 +42,20 @@ export const AlchemyAccountContext = createContext<
 >(undefined);
 
 export type AlchemyAccountsUIConfig = {
-  auth?: AuthCardProps & { addPasskeyOnSignup?: boolean };
+  header?: ReactNode;
```

Comment:
imo it doesn't make sense to nest this within `auth` anymore, since there is no real distinction between the UI and "auth" - it's all auth UI config... if there's a forward-looking reason I don't know about, happy to revert, but it felt weird to be spread across top-level and a nested object (and introduced type weirdness where the top level could be present but not the inner auth object)

Reply:
yea the initial reason for this was to support the case where we might have like sign message pop-ups or something. If you're using `useSendUserOperation` or `useSignMessage` you might get a sign message or tx pop-up if you opt in to that